packages/gatsby-theme-patternfly-org/components/footer/footer.js

```diff
@@ -156,11 +157,7 @@ export class Footer extends React.Component {
             <GridItem sm={12} md={4}>
               <Grid className="pf-u-pt-xl ws-org-pfsite-l-footer-column">
                 <GridItem className="pf-u-px-xl">
-                  <img
-                    src={patternflyLogo}
-                    alt="PatternFly logo"
-                    className="ws-org-pfsite-footer-menu-about-logo pf-u-pb-md"
-                  />
+                  <a class="pf-c-page__header-brand-link pf-c-brand ws-org-pfsite-footer-menu-about-logo pf-u-pb-md" href="/"><Brand src={logo} alt="Patternfly Logo" /></a>
```

Comment:
It appears that we already have this logo in packages/gatsby-theme-patternfly-org/layouts/logo.svg?

Reply:
Fixed